api/src/opentrons/file_runner/json_command_translator.py

```diff
@@ -1,3 +1,4 @@
+"""Translation of JSON protocol atomic commands into ProtocolEngine commands."""
```

Comment:
Nitpick:

```suggestion
"""Translation of JSON protocol commands into ProtocolEngine commands."""
```

To avoid future confusion with atomicity in other senses. Since the Python Protocol API no longer calls these "atomic commands," I think the term is deprecated?